src/ext/framework/help_commands.rs

```diff
@@ -21,18 +33,34 @@ pub fn with_embeds(ctx: &Context,
         let name = args.join(" ");
 
         for (group_name, group) in groups {
-            let mut found: Option<(&String, &Command)> = None;
+            let mut found: Option<(&String, &Arc<Command>)> = None;
 
             if let Some(ref prefix) = group.prefix {
                 for (command_name, command) in &group.commands {
-                    if name == format!("{} {}", prefix, command_name) {
-                        found = Some((command_name, command));
+                    if name == format!("{} {}", prefix, command_name) || name == command_name.to_owned() {
```

Comment:
You can compare Strings with string slices; you shouldn't need to create an owned String here